internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -81,9 +81,9 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 
 	var clusterResources aws.ClusterResources
 	if kopsMetadata.ChangeRequest.VPC != "" && provisioner.params.UseExistingAWSResources {
-		clusterResources, err = awsClient.GetVpcResourcesByVpcID(kopsMetadata.ChangeRequest.VPC, logger)
+		clusterResources, err = awsClient.ClaimVPC(kopsMetadata.ChangeRequest.VPC, cluster, provisioner.params.Owner, logger)
 		if err != nil {
-			return err
+			return errors.Wrap(err, "Couldn't claim VPC")
```

Comment:
nit, lowercase letters for wrapped error returns
```suggestion
			return errors.Wrap(err, "couldn't claim VPC")
```